MAUI/Segmented-Control/right-to-left.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Right To Left FlowDirection in .NET MAUI Segmented control | Syncfusion
+description: Learn about Right To Left support in Syncfusion .NET MAUI Segmented (SfSegmented) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfSegmentedControl
+documentation: ug
+---
+ 
+# Right To Left Flow Direction in .NET MAUI Segmented (SfSegmented) control
+
+The `SfSegmentedControl` supports changing the flow direction of the text to the right-to-left direction by setting the `FlowDirection` to `RightToLeft`.
```

Comment:
The `SfSegmentedControl` supports changing the flow direction of items rendering in the right-to-left direction by setting the `FlowDirection` to `RightToLeft`.

Reply:
content changed